sycl/include/CL/sycl/detail/cg_types.hpp

```diff
@@ -245,9 +245,9 @@ class HostTask {
 template <class KernelType, class KernelArgType, int Dims, typename KernelName>
 class HostKernel : public HostKernelBase {
   using IDBuilder = sycl::detail::Builder;
-  KernelType MKernel;
 
 public:
+  KernelType MKernel;
```

Comment:
This doesn't seem to be backward compatible change.
Please, add getter/setter methods or declare `friend` class or method.

Reply:
`friend` is added for `sycl::handler`. `MKernel` is brought back to `private`.